### ISSUE TITLE
Command failed with KeyError: !spoiler Some Disassembly Required

### ISSUE THREAD

**Issue by vorpal-buildbot** [PennyDreadfulMTG/Penny-Dreadful-Tools#1639]
```
Traceback (most recent call last):
  File "/home/discord/Penny-Dreadful-Discord-Bot/discordbot/command.py", line 50, in handle_command
    await method(Commands, bot, message.channel, args, message.author)
  File "/home/discord/Penny-Dreadful-Discord-Bot/discordbot/command.py", line 373, in spoiler
    fetcher.internal.store(sfcard['image_uri'], imagepath)
KeyError: 'image_uri'

```

Reported on Discord by Yasuhiko#9168

**Comment by bakert** [comment]
Is this related to the bug I fixed in StackIt where Scryfall moved to a dict of image uris for various sizes instead of single uri? Probably.

**Comment by bakert** [comment]
It was/is that. Fix is 5cca78f.
